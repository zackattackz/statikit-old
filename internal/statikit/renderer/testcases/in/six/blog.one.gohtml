{{.FileSub.Head}}
Post title {{.Data.Title}}