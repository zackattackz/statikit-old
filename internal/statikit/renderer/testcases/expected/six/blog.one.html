<head><title>Hello World</title></head>
Post title One